includes/wf_crm_webform_postprocess.inc

```diff
@@ -125,7 +125,7 @@ class wf_crm_webform_postprocess extends wf_crm_webform_base {
 
     $this->fillDataFromSubmission();
     //Fill Custom Contact reference fields.
-    $this->fillContactRefs(NULL, 0, TRUE);
+    $this->fillContactRefs(TRUE, NULL, 0);
```

Comment:
The reason I suggested adding the new param to the beginning is so you could omit default params.
```suggestion
    $this->fillContactRefs(TRUE);
```

Reply:
ah yes, sorry. Have updated.